tests/periph_gpio_arduino/main.sketch

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2017 Tom Keddie
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Manual test application for GPIO peripheral drivers using
+ *              Arduino pin numbering
+ *
+ * @author      Tom Keddie <github@bronwenandtom.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "shell.h"
+#include "periph/gpio.h"
+
+static int get_digital_pin(const char* pinstring, int* pin)
```

Comment:
It's a good idea to check whether the given parameter makes sense. Currently everything is accepted. It would be niced if you test whether the pinstring is a numerical character (since A pins are anyway not accepted).